mojo/public/cpp/system/invitation.cc

```diff
@@ -187,28 +189,45 @@ void OutgoingInvitation::Send(OutgoingInvitation invitation,
                  MOJO_SEND_INVITATION_FLAG_NONE, error_callback, "");
 }
 
+// static
+void OutgoingInvitation::Send(OutgoingInvitation invitation,
+                              base::ProcessHandle target_process,
+                              PlatformChannelServerEndpoint server_endpoint,
+                              const ProcessErrorCallback& error_callback) {
+  SendInvitation(std::move(invitation.handle_), target_process,
+                 server_endpoint.TakePlatformHandle(),
+                 MOJO_INVITATION_TRANSPORT_TYPE_CHANNEL_SERVER,
```

Comment:
It seems that we have to use MOJO_INVITATION_TRANSPORT_TYPE_CHANNEL here,
and MOJO_INVITATION_TRANSPORT_TYPE_CHANNEL_SERVER below.
The places of both of them has been switched I guess.

Reply:
There are 2 types of static wrapper functions in upstream. the parameters are different.

1. Send(..., PlatformChannelEndpoint channel_endpoint, ...)
  => Pass MOJO_INVITATION_TRANSPORT_TYPE_CHANNEL
2. Send(..., PlatformChannelServerEndpoint server_endpoint, ...)
  => Pass MOJO_INVITATION_TRANSPORT_TYPE_CHANNEL_SERVER

Reply:
I see. Thank you for your explanation.
With that explanation now I see that in the function |OutgoingInvitation::Send| below located in #if defined(CASTANETS) macro,
SendInvitation(...,
                 server_endpoint.TakePlatformHandle(),
                 MOJO_INVITATION_TRANSPORT_TYPE_CHANNEL,...)
looks odd.
If you are intending to represent client socket here,
then it would be better to rename the argument |PlatformChannelEndpoint server_endpoint|
to |PlatformChannelEndpoint channel_endpoint|.

Reply:
My mistake:) I'll update the parameter name.